app-info/endlessos_forum.xml

```diff
@@ -0,0 +1,23 @@
+<component type="web-application">
+    <launchable type="url">https://community.endlessos.com/</launchable>
+    <url type="homepage">https://community.endlessos.com/</url>
+    <project_license>GPL-2.0-or-later</project_license>
+    <metadata_license>FSFAP</metadata_license>
+    <name>Endless</name>
+    <id>org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec.desktop</id>
+    <icon type="remote" width="512" height="512">https://uploads-community-endlessos-com.s3.dualstack.us-west-2.amazonaws.com/optimized/1X/8eff890b1bc1880b0077955aba6e2e76223a1365_2_512x512.jpg</icon>
```

Comment:
The screenshots are great, but I think you need to reference them from the XML or they won't do anything. You want a section that looks like:

```
    <screenshots>
      <screenshot type="default">
        <image width="540" height="720">https://pinafore.social/screenshot-540-720-1.png</image>
      </screenshot>
      <screenshot type="default">
        <image width="540" height="720">https://pinafore.social/screenshot-540-720-2.png</image>
      </screenshot>
    </screenshots>
```

I don't know what the width and height of the screenshots you took are. Hopefully you can find those without too much effort.

The tricky part is the URL. This repo gets published in a way where the `s3` directory is available at https://d3lapyynmdp1i9.cloudfront.net/. So, I think we'll want to reorganize the screenshots a bit. For now, ignore the top level screenshots directory. It's an implementation detail of gnome-software and I think I'll just write something to automate handling it.

What we actually care about is the s3/screenshots directory. Notice that the screenshots are organized by app there. What I would do is create a folder at s3/screenshots/org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec. Then move all of your screenshots there but with simpler names like `screenshot1.png`, `screenshot2.png`, etc.

Does that make sense?

Reply:
I see what you mean about the URL being the tricky part. I relocated the screenshots to your suggested place and tried using the permalinks for the URLs but when I run the 
> gnome-software --show-metainfo
command it doesn't load the screenshots.

Reply:
Sorry, I wasn't very clear about the URL. It's hard to test because the screenshots in your PR won't be uploaded until we merge your PR. What's going to happen is that the s3 directory is published at https://d3lapyynmdp1i9.cloudfront.net/. So, an example screenshot for this app would be https://d3lapyynmdp1i9.cloudfront.net/screenshots/org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec/screenshot1.png.

That doesn't help you a lot for testing how things look. For that you'll have to temporarily mangle the URLs. One way to do this is to run a local HTTP server that mimics it. For example, you can run `python3 -m http.server --directory s3`. Then you can temporarily change the URL to http://127.0.0.1:8000/screenshots/org.gnome.Software.WebApp_5c818dd1c0c3cad43709490cb72217f0d08912ec/screenshot1.png. Hopefully gnome-software let's you do a non-https URL.